### PR TITLE
Fix missing spaces in system prompt of Llama2 tokenizer

### PR DESCRIPTION
**Pull request by chenjoya**
# What does this PR do?

This PR fixes a typo  in system prompt of Llama2 tokenizer. There are missing spaces compared to https://github.com/facebookresearch/llama/blob/main/llama/generation.py

Thank you.

### CODE REVIEW

**Comment by sgugger** [comment]
cc @ArthurZucker

**Comment by ArthurZucker** [comment]
Hey thanks, a similar PR is also #24935. Same comment would apply here!

File: src/transformers/models/llama/tokenization_llama.py [hunk 1]
```diff
@@ -53,11 +53,11 @@
 B_SYS, E_SYS = "<<SYS>>\n", "\n<</SYS>>\n\n"
 
 # fmt: off
-DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your\
-answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure\
+DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your \
+answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure \
 that your responses are socially unbiased and positive in nature.
```
Comment (ArthurZucker) [inline src/transformers/models/llama/tokenization_llama.py:58]:
```suggestion
answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure
\ that your responses are socially unbiased and positive in nature.
```

File: src/transformers/models/llama/tokenization_llama_fast.py [hunk 2]
```diff
@@ -40,11 +40,11 @@
 B_SYS, E_SYS = "<<SYS>>\n", "\n<</SYS>>\n\n"
 
 # fmt: off
-DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your\
-answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure\
+DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your \
+answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure \
 that your responses are socially unbiased and positive in nature.
```
Comment (ArthurZucker) [inline src/transformers/models/llama/tokenization_llama_fast.py:45]:
```suggestion
answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure
\ that your responses are socially unbiased and positive in nature.
```

**Review by ArthurZucker — COMMENTED** [review]
These should be good to make the ci go green!

**Comment by chenjoya** [comment]
Hi, thank you so much for your help. Seems it still cannot go green. I am not very familiar with it ... Could you give me some guidance? ;)

File: src/transformers/models/llama/tokenization_llama.py [hunk 3]
```diff
@@ -53,11 +53,11 @@
 B_SYS, E_SYS = "<<SYS>>\n", "\n<</SYS>>\n\n"
 
 # fmt: off
-DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your\
-answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure\
-that your responses are socially unbiased and positive in nature.
+DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your \
+answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure
+\ that your responses are socially unbiased and positive in nature.
```
Comment (ArthurZucker) [inline src/transformers/models/llama/tokenization_llama.py:59]:
```suggestion
DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your \
answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure\
 that your responses are socially unbiased and positive in nature.

```
`ruff` does not like the escape sentence being here (was automatically formatted by `make style` and ruff adds `\\`.

File: src/transformers/models/llama/tokenization_llama_fast.py [hunk 4]
```diff
@@ -40,11 +40,11 @@
 B_SYS, E_SYS = "<<SYS>>\n", "\n<</SYS>>\n\n"
 
 # fmt: off
-DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your\
-answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure\
-that your responses are socially unbiased and positive in nature.
+DEFAULT_SYSTEM_PROMPT = """You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe. Your \
+answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure
+\ that your responses are socially unbiased and positive in nature.
```
Comment (ArthurZucker) [inline src/transformers/models/llama/tokenization_llama_fast.py:45]:
```suggestion
answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure\
 that your responses are socially unbiased and positive in nature.
```

**Review by ArthurZucker — COMMENTED** [review]
Okay this should be it!

**Review by ArthurZucker — APPROVED** [review]
Thanks a lot for fixing

**Review by sgugger — APPROVED** [review]
Thanks for fixing!

**Pull request merged**